<?xml version="1.0" encoding="UTF-8"?><testrun name="CreatureTests" project="Homework1" tests="4" started="4" failures="1" errors="0" ignored="0">
  <testsuite name="tests.CreatureTests" time="0.024">
    <testcase name="toStringIncludesClass" classname="tests.CreatureTests" time="0.003"/>
    <testcase name="FlyCantEatCreatures" classname="tests.CreatureTests" time="0.016">
      <failure>java.lang.AssertionError: Expected exception: Things.Fly$IllegalFood
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:33)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
    <testcase name="BellyStartsEmpty" classname="tests.CreatureTests" time="0.0"/>
    <testcase name="eatingStores" classname="tests.CreatureTests" time="0.001"/>
  </testsuite>
</testrun>
